cyral/resource_cyral_policy_rule.go

```diff
@@ -324,19 +327,18 @@ func resourcePolicyRuleUpdate(ctx context.Context, d *schema.ResourceData, m int
 	return resourcePolicyRuleRead(ctx, d, m)
 }
 
-func resourcePolicyRuleDelete(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-	log.Printf("[DEBUG] Init resourcePolicyRuleDelete")
-	c := m.(*client.Client)
-
-	url := fmt.Sprintf("https://%s/v1/policies/%s/rules/%s", c.ControlPlane, d.Get("policy_id").(string), d.Id())
-
-	if _, err := c.DoRequest(url, http.MethodDelete, nil); err != nil {
-		return createError("Unable to delete policy rule", fmt.Sprintf("%v", err))
+func deletePolicyRule() ResourceOperationConfig {
```

Comment:
Nit: What do you think of naming this function as `policyRuleDeleteConfig`? I think it reflects better its purpose. Just a suggestion though.
```suggestion
func policyRuleDeleteConfig() ResourceOperationConfig {
```